tests/foreman/ui/test_contentview.py

```diff
@@ -1830,115 +1834,406 @@ def test_positive_restart_publish_via_dynflow(self):
     # ROLES TESTING
     # All this stuff is speculative at best.
 
-    @stubbed()
     @run_only_on('sat')
     @tier2
     def test_positive_admin_user_actions(self):
-        # Note:
-        # Obviously all of this stuff should work with 'admin' user
-        # but these tests require creating a user with admin permissions
-        # for Content Views
-        # Dev note: none of this stuff is integrated with foreman rbac yet
-        # As such, all variations in here subject to change.
-        # Variations:
-        #  * Read, Modify, Delete, Promote Publish, Subscribe
-        """attempt to view content views
+        """Attempt to manage content views
 
         @id: c4d270fc-a3e6-4ae2-a338-41d864a5622a
 
-        @setup: create a user with the Content View admin role
+        @steps:
+        with global admin account:
 
-        @assert: User with admin role for content view can perform all
-        Variations above
+        1. create a user with all content views permissions
+        2. create lifecycle environment
+        3. create 2 content views (one to delete, the other to manage)
 
-        @caseautomation: notautomated
+        @setup: create a user with all content views permissions
 
+        @assert: The user can Read, Modify, Delete, Publish, Promote the
```

Comment:
@oshtaier Note the user is not a global admin, we have created a user without permissions, and added only the required permission to work with the content view, publish promote to environment and view environments in order to publish/promote.